src/components/ErrorBoundary/index.tsx

```diff
@@ -1,66 +1,157 @@
 import { Trans } from '@lingui/macro'
 import * as Sentry from '@sentry/react'
-import React, { PropsWithChildren } from 'react'
+import { ButtonLight, ButtonPrimary } from 'components/Button'
+import { ChevronUpIcon } from 'nft/components/icons'
+import { useIsMobile } from 'nft/hooks'
+import React, { PropsWithChildren, useState } from 'react'
+import { Copy } from 'react-feather'
 import styled from 'styled-components/macro'
 
-import { ExternalLink, ThemedText } from '../../theme'
-import { AutoColumn } from '../Column'
-import { AutoRow } from '../Row'
+import { CopyToClipboard, ExternalLink, ThemedText } from '../../theme'
+import { Column } from '../Column'
 
 const FallbackWrapper = styled.div`
   display: flex;
-  flex-direction: column;
-  width: 100%;
-  align-items: center;
-  z-index: 1;
+  width: 100vw;
+  height: 100vh;
 `
 
 const BodyWrapper = styled.div<{ margin?: string }>`
+  width: min(100%, 500px);
+  margin: auto;
   padding: 1rem;
+`
+
+const SmallButtonPrimary = styled(ButtonPrimary)`
+  width: auto;
+  font-size: 16px;
+  padding: 10px 16px;
+  border-radius: 12px;
+`
+
+const SmallButtonLight = styled(ButtonLight)`
+  font-size: 16px;
+  padding: 10px 16px;
+  border-radius: 12px;
+`
+
+const StretchedRow = styled.div`
+  display: flex;
+  gap: 24px;
+
+  > * {
+    display: flex;
+    flex: 1;
+  }
+`
+
+const Code = styled.code`
+  font-weight: 300;
+  font-size: 12px;
+  line-height: 16px;
+  word-wrap: break-word;
   width: 100%;
+  color: ${({ theme }) => theme.textPrimary};
+  font-family: courier, courier new, serif;
```

Comment:
feels like this should live as a variable somewhere

Reply:
Why not, I can put it under theme as `codeFont`, albeit we're only using it in this component.

Reply:
i think we would potentially use it elsewhere

Reply:
Actually FWIW Figma references a different font, but I didn't justify pulling it for a single component at this point.

Reply:
Design team is aware and ok